packages/devtools_app/lib/src/shared/http/_http_cookies.dart

```diff
@@ -137,37 +137,6 @@ class Cookie {
     parseAttributes();
   }
 
-  String toString() {
```

Comment:
This one is strange, either the annotation is missing or I don't understand how it supposed to be used 🤔